FeatureIds.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Lombiq.HelpfulExtensions
+{
+    public static class FeatureIds
+    {
+        private const string FeatureIdPrefix = "Lombiq.HelpfulExtensions.";
+
+        public const string Lombiq_HelpfulExtensions_Flows = FeatureIdPrefix + "Flows";
+        public const string Lombiq_HelpfulExtensions_Widgets = FeatureIdPrefix + "Widgets";
+        public const string Lombiq_HelpfulExtensions_ContentTypes = FeatureIdPrefix + "ContentTypes";

```

Comment:
We don't need to prefix feature ID consts' names as the class itself is already within the module. So just Flows, Widgets...

Reply:
Based on my experience if you are using these constants in other module's manifest files (i.e. defining dependencies) it is not obvious where this is coming from. Still you could define an alias in usings (e.g. `using HelpfulLibrariesFeatureIds = Lombiq.HelpfulExtensions.FeatureIds;`) though I find the readibility much better if instead of using these we define the constants with the full feature names and adding them in other manifests as static (e.g. `using static Lombiq.HelpfulExtensions.FeatureIds`) and in the Dependencies list just use them like

```
Dependencies = new[] {
  "OrchardCore.Alias",
  "OrchardCore.Flows",
  Lombiq_HelpfulExtensions_Flows,
  MyCompany_OtherModule_OtherFeature
}
```

So we don't break the feature name pattern even if we are using constants.

Reply:
You don't need an alias, you can just use the full namespace, should there be a lot of such consts being used in the same manifest: `Lombiq.HelpfulExtensions.FeatureIds.Flows`. Namespaces are the mechanics to deal with name clashes or confusion, we don't need to duplicate them in field names.

Reply:
Okey-dokey.